guix_studio/build/vs_2019/msix_package_project/Package.appxmanifest

```diff
@@ -9,12 +9,13 @@
   IgnorableNamespaces="uap rescap">
 
   <Identity
-    Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"
-    Version="6.4.0.0"  Name="Microsoft.AzureRTOSGUIXStudio"/>
+    Name="Eclipse.EclipseThreadXTraceX"
+    Publisher="CN=Eclipse Foundation, O=Eclipse Foundation, L=Ottawa, S=Ontario, C=CA"
```

Comment:
I believe changing the publisher identify will require updating the certificate. Otherwise the app might not start.

Reply:
reverted the publisher.